src/custom-ops.ts

```diff
@@ -0,0 +1,9 @@
+import { filter, throwIfEmpty } from 'rxjs/operators';
+import { pipe } from 'rxjs';
+
+export function throwIfFalse(predicate, error_func) {
```

Comment:
Can we store this in `src/app/shared/lib` instead?

Reply:
Done